dotCMS/src/main/java/com/dotcms/contenttype/business/ContentTypeFactoryImpl.java

```diff
@@ -683,4 +683,21 @@ public void validateFields(ContentType type) {
     
   }
 
+ @Override
+ public void updateModDate(ContentType type) throws DotDataException {
+	 ContentTypeBuilder builder =
+		        ContentTypeBuilder.builder(type).modDate(DateUtils.round(new Date(), Calendar.SECOND));
+	 type = builder.build();
```

Comment:
Can we round this to seconds?  Otherwise the milliseconds can break the equals method

```
    ContentTypeBuilder builder =
        ContentTypeBuilder.builder(saveType).modDate(DateUtils.round(new Date(), Calendar.SECOND));
```